### ISSUE TITLE
Tooling: Allow commands to have aliases

### ISSUE THREAD

**Issue by AaronFeledy** [lando/lando#2026]
As a user who frequently implements and changes my custom tooling, I would like the ability to add aliases to commands. This allows for shorthand versions of tooling commands where something like `lando download-remote-files` could also be executed by typing `lando dlf`. This also allows for retooling things but allowing users to continue to use legacy commands to execute new tooling.

Lando could provide an `alias` key to the tooling config, allowing one or more aliases to be defined. `lando --help` could then show aliases along with the fully qualified tooling names.

**Comment by serundeputy** [comment]
Hello @AaronFeledy !

I think you can satisfy this with:

```yml
tooling:
  dlf:
    service: appserver # or whateva
    description: Get dem files
    cmd: download-remote-files
```

Maybe that is good enough to satisfy the use case? i.e, you'd have two tooling entries one for the long name and one short  (i didn't test it, but i think it'll work)

**Comment by AaronFeledy** [comment]
Hi @serundeputy!

Unfortunately, you can't chain together Lando commands in that way. Such a feature would definitely satisfy this use case and many others that I have. It's another feature request I've been putting off making an issue for. Maybe I'll do that now.

**Comment by AaronFeledy** [comment]
If implemented, #2107 could allow for the workaround described above.
